src/main/kotlin/no/nav/syfo/personoppgave/PersonOppgaveService.kt

```diff
@@ -44,7 +44,38 @@ class PersonOppgaveService(
         }
     }
 
-    fun behandle(personOppgave: PersonOppgave, veilederIdent: String) {
+    fun behandlePersonOppgaver(
+        personoppgaver: List<PersonOppgave>,
+        veilederIdent: String,
+    ) {
+        val now = OffsetDateTime.now().toLocalDateTimeOslo()
+        val updatedPersonOppgaver = personoppgaver.map {
+            it.copy(
```

Comment:
Ser bra ut. Kan evt vurdere å skille ut oppdatering av disse feltene i egen metode siden man nå gjør dette to steder i her i servicen.

Reply:
Ikke dumt, se siste forslag 👍🏼